src/QtUtils/CMakeLists.txt

```diff
@@ -9,9 +9,10 @@ add_library(QtUtils STATIC)
 
 target_compile_options(QtUtils PRIVATE ${STRICT_COMPILE_FLAGS})
 target_include_directories(QtUtils PUBLIC include/)
-target_link_libraries(QtUtils PUBLIC OrbitBase OrbitGl Qt5::Core CONAN_PKG::abseil)
+target_link_libraries(QtUtils PUBLIC OrbitBase OrbitGl Qt5::Core CONAN_PKG::abseil GTest::GTest)
```

Comment:
This means GTest is now linked in our main Orbit binary? Maybe a QtTestUtils component is a better idea?

Reply:
It's not really an issue because GTest is a static library. The linker will strip all the unneeded symbols.

It would be a problem if we wanted to use QtUtils in some place where GTest is not available, but since GTest is needed everywhere (all platforms, all modules) that's not really an issue.

Reply:
Okay then we leave it as is